test/functional/rpc_blockchain.py

```diff
@@ -307,6 +309,16 @@ def assert_waitforheight(height, timeout=2):
         assert_waitforheight(current_height)
         assert_waitforheight(current_height + 1)
 
+    def _test_getblockbyheight(self):
+        self.log.info("Test getblockbyheight")
+        node = self.nodes[0]
+        node.add_p2p_connection(P2PInterface())
+
+        self.log.info("Test that getblockbyheight is getting block 20")
+        assert_equal(node.getblock(node.getblockhash(20)), node.getblockbyheight(20))
+        self.log.info("The block is valid")
```

Comment:
I'd recommend removing the logging on 317 and 319 because it's not needed and will clutter the console outside.  Other than that, things look good.

Reply:
Good point, removed!